### PR TITLE
[BAEL-5406] IncompatibleClassChange Error

### PR DESCRIPTION
**Pull request by bhaskar16**
### [BAEL-5406] IncompatibleClassChange Error

### CODE REVIEW

**Comment by lor6** [comment]
retest this please

File: libraries-bytecode/src/main/java/com/baeldung/incompatibleclasschange/libraries/modified/Dinosaur.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.baeldung.incompatibleclasschange.libraries.modified;
+
+public class Dinosaur {
+    public static void species(String sp) {
+	if(sp == null) {
+		System.out.println("I am a generic Dinosaur");
```
Comment (ashleyfrieze) [inline libraries-bytecode/src/main/java/com/baeldung/incompatibleclasschange/libraries/modified/Dinosaur.java:6]: The spacing in this file is incorrect. It uses tabs, where we should be using spaces.

Comment (bhaskar16) [reply libraries-bytecode/src/main/java/com/baeldung/incompatibleclasschange/libraries/modified/Dinosaur.java]: Fixed.

File: libraries-bytecode/pom.xml [hunk 2]
```diff
@@ -40,6 +40,21 @@
             <artifactId>asm-util</artifactId>
             <version>${asm.version}</version>
         </dependency>
+
+<!--        <dependency>-->
```
Comment (ashleyfrieze) [inline libraries-bytecode/pom.xml:44]: Maybe we can put a comment above here to explain what to uncomment etc.

**Review by ashleyfrieze — CHANGES_REQUESTED** [review]

File: libraries-bytecode/pom.xml (hunk 2 above)
Comment (bhaskar16) [reply libraries-bytecode/pom.xml]: Understood. Added

File: libraries-bytecode/src/main/java/com/baeldung/incompatibleclasschange/libraries/original/Dinosaur.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package com.baeldung.incompatibleclasschange.libraries.original;
+
+public class Dinosaur {
+    public void species(String sp) {
+	if(sp == null) {
+		System.out.println("I am a generic Dinosaur");
```
[the remark this answers is not in the document]
Comment (bhaskar16) [reply libraries-bytecode/src/main/java/com/baeldung/incompatibleclasschange/libraries/original/Dinosaur.java]: Added comment

**Pull request merged**